### ISSUE TITLE
Any auto clear cache option?

### ISSUE THREAD

**Issue by channchetra** [terrylinooo/cache-master#16]
I want auto clear cache and preloading cache anyone know how todo? only schedule clear cache should be fine.
thank you.

**Comment by prakash4mail** [comment]
@channchetra  As much code as i have seen, it resets every 24 hours and also it clears when the post is updated, but feel free to further look into the code. @terrylinooo can confirm too

**Comment by channchetra** [comment]
Thank every much @prakash4mail by the way example my Site has only few page and post and only home page is dynamically by get data from external API Every 5 minute so I want clear cache every 5 minute and caching back. if any way can do please help. if can do by system level it's ok to do.

**Comment by prakash4mail** [comment]
@channchetra  you can go to settings of the plugin and use exclusion page to do that., 
try this: 1. add to exclusion; 2. enable benchmark; 3. open page in incognito and ctrl+F5 to reload without cache ; and see if cache is disabled,;  5. then disable benchmark and you are good to go.

**Comment by channchetra** [comment]
@prakash4mail Yes now day I'm excluded that page so when we open that page it's to slow. I want can't cache that page to but every 10 or 30 minute I want auto clear that cache and preload it cache back. so we can boost preferment for our visitor. anyway can do? if can do with server cronjob it ok too. thank every much for your time 😊

**Comment by channchetra** [comment]
so there are no solution?

**Comment by channchetra** [comment]
so there are no author support?

**Comment by yannicklin** [comment]
Is this setting not working? @channchetra 
<img width="693" alt="image" src="https://user-images.githubusercontent.com/1244339/188072479-71a2f470-981a-4980-9556-a1065b816eea.png">

**Comment by channchetra** [comment]
@yannicklin I don't know what it mean never touch that option. by the way I gave up with this plugin there're no support from author here.
